osv/bare64/main.dpl

```diff
@@ -0,0 +1,4123 @@
+module osv.bare64.main:
```

Comment:
No need for a duplicate `main.dpl` if nothing changes

Reply:
Resolved by runtime/arch split.